variantgrid/static_files/default_static/js/vc_form.js

```diff
@@ -969,14 +969,28 @@ const VCForm = (function() {
 
             appendLabelHeadingForKey(SpecialEKeys.ASSERTION_METHOD, true, "Method")
 
+            let label = "Condition";
+            if (alleleOriginBucket === "S") {
+                label = "Cancer";
+            }
+
+            let conditionElement;
             if (this.record.resolved_condition) {
-                let label = "Condition";
-                if (alleleOriginBucket === "S") {
-                    label = "Cancer";
+                conditionElement = VCForm.format_condition(this.record.resolved_condition);
+            } else {
+                let condition = this.value(SpecialEKeys.CONDITION);
+                let condition_url = Urls.condition_matching(this.condition_resolution);
+
+                if (this.isEditMode() && this.condition_matching_is_view_enabled === 'true') {
```

Comment:
Reminder if this is changed to a proper boolean you wont need to compare it to the string 'true'